source/concepts/applications.rst

```diff
@@ -285,12 +283,15 @@ users, and how to render the molecules.
 Frames
 ~~~~~~
 
-The :ref:`trajectory service <trajectory-service>` allows to stream snapshots
-of arbitrary data to clients. Each snapshot is described in a :ref:`FrameData
-<frame-description>` which contains a key-value map of protobuf `Values
-<https://protobuf.dev/reference/protobuf/google.protobuf/#value>`_ and one of
-homogeneous arrays. Here, we define a set of keys and data format to describe
-the semantics of molecular systems.
+The trajectory application uses the :ref:`trajectory service <trajectory-service>`,
+which allows a server to stream snapshots of arbitrary data to clients. Each snapshot is
+described in a :ref:`FrameData <frame-description>` object, which contains:
+
+* a key-value map of protobuf `Values <https://protobuf.dev/reference/protobuf/google.protobuf/#value>`_
+* a key-value map of homogeneous arrays
+
+Here, we define a set of keys and data formats to describe the semantics of
+molecular systems.
 
 .. note::
```

Comment:
This note could do with rewording, maybe "A server using this set of keys can implement keys from another application as well. For instance, a server implementing the [iMD application](https://github.com/IRL2/nanover-docs/pull/155/files#imd-application) can implement both the trajectory-specific keys and [iMD-specific keys](https://github.com/IRL2/nanover-docs/pull/155/files#imd-framedata-keys).